addons/udes_stock/models/stock_picking.py

```diff
@@ -309,6 +310,41 @@ def action_done(self):
         all_picks = picks | picks.mapped("u_next_picking_ids")
         all_picks.with_context(lock_batch_state=False)._trigger_batch_state_recompute()
 
+        # If operation type does not allow multiple lots in a single transfer then check for number
```

Comment:
Can we move this to another function? so we don't make action_done get bigger an harder to understand.

Reply:
Good point. Have moved this logic into its own `assert_not_lot_restricted` function.